src/IconButton/_IconButton.scss

```diff
@@ -15,28 +15,41 @@ $btn-icon-diameter-inline: calc(#{$line-height-base}em + 0.1em) !default;
 
 $btn-icon-bg: transparent !default;
 $btn-icon-accent-color: white !default;
+$btn-active-bg: $gray-700 !default;
 
 @mixin btn-icon-size($icon-size) {
     width: $icon-size;
     height: $icon-size;
 }
 
-@mixin btn-icon-variant($icon-color, $accent-color) {
-    background-color: $btn-icon-bg;
-    color: $icon-color;
-
+@mixin btn-icon-variant($icon-color, $accent-color, $active-state: false) {
     &.hover,
-    &:hover {
+    &:hover,
+    &.active,
```

Comment:
Question: Should the hover state be the same as the active selected state? I know it came from the original hover for the button component, so I understand if we made this decision to be consistent, but it kind of looks weird when one is selected and you're hovering over the other option

Reply:
good question! I’ll circle back with Gabe on if we need to revisit this portion, thanks for the query!